runtime/jcl/common/stdinit.c

```diff
@@ -187,6 +187,23 @@ standardInit( J9JavaVM *vm, char *dllName)
 			result = (jint)initializeSystemThreadGroup(vm, (JNIEnv *)vmThread);
 			if (JNI_OK != result) {
 				goto _fail;
+			} else {
+#if JAVA_SPEC_VERSION >= 15
+				JNIEnv *env = (JNIEnv *)vmThread;
+				jclass clz = (*env)->FindClass(env, "jdk/internal/loader/NativeLibraries");
+				jmethodID mid = NULL;
+				if (NULL == clz) {
+					goto _fail;
+				}
+				mid = (*env)->GetStaticMethodID(env, clz, "load", "(Ljdk/internal/loader/NativeLibraries$NativeLibraryImpl;Ljava/lang/String;ZZ)Z");
+				if (NULL == mid) {
+					goto _fail;
+				}
+				vm->nativeLibrariesLoadMethodID = (UDATA) mid;
+				Trc_JCL_init_nativeLibrariesLoadMethodID(vmThread, vm->nativeLibrariesLoadMethodID);
+#else /* JAVA_SPEC_VERSION >= 15 */
```

Comment:
Not strictly necessary - it will already be 0.

Reply:
Yeah, that's right. Removed the zero setting.